src/jaxsim/rbda/contacts/rigid.py

```diff
@@ -295,60 +256,58 @@ def compute_contact_forces(
                 Optional `(n_joints,)` vector of joint forces.
 
         Returns:
-            A tuple containing the contact forces.
+            A tuple containing as first element the computed contact forces.
         """
 
         # Initialize the model and data this contact model is operating on.
         # This will raise an exception if either the contact model or the
         # contact parameters are not compatible.
         model, data = self.initialize_model_and_data(model=model, data=data)
+        assert isinstance(data.contacts_params, RigidContactsParams)
 
-        # Import qpax just in this method
+        # Import qpax privately just in this method.
         import qpax
 
-        link_forces = (
-            link_forces
+        link_forces = jnp.atleast_2d(
+            jnp.array(link_forces, dtype=float).squeeze()
             if link_forces is not None
             else jnp.zeros((model.number_of_links(), 6))
-        )
+        ).astype(float)
 
-        joint_force_references = (
-            joint_force_references
+        joint_force_references = jnp.atleast_1d(
+            jnp.array(joint_force_references, dtype=float).squeeze()
```

Comment:
Same as above, probably redundant

```suggestion
            jnp.array(joint_force_references).squeeze()
```

Reply:
Done in https://github.com/ami-iit/jaxsim/compare/7b2fc3bfe40a8b4998b768df2a2b0f5d959a8e2a..f2e613f4fafe2288f0e4114ed64dbced4a160f39